README.md

```diff
@@ -1,6 +1,16 @@
 # ob-openapi-taxonomy
-Master Files for OB OpenAPI
+**Master Files for OB OpenAPI** (Includes: Master Solar Taxonomy File and the OB OpenAPI Master File)
 
-Includes the:
-Master Solar Taxonomy File and the OB OpenAPI Master File
+The Orange Button Solar Taxonomy is defined within **Master-OB-OpenAPI.json**, if you are here looking at this page, this file is the one you will most likely be interested in.
 
+For those with git skills, please feel free to fork this repository and then submit a pull request with your suggested changes.  For those without git skills but still wanting to contribute ideas or changes to this file, please feel free to access Cliff's fork of this repository which is available via a google drive file link [here](https://drive.google.com/file/d/1-iPr4dQ4RYT_FgLC8jW6s1wjga4O38ox/view?usp=sharing) (for those new to working with the Orange Button team, you will likely be prompted to request access before you can begin editing it, please do request this access if you are interested in contributing). <em>**Note to Jan and Cliff etc, I think this section needs to be tweaked to explain the exact approach you are using to leverage Cliff's fork via a google local drive folder (I think it involves installing "Google Drive" on your local machine and then syncing with the google drive so that this json file appears on your machine and you can edit it as if it was a local file right?)**</em>
+
+If you would prefer to visually explore and edit a copy of this json file (rather than editing the raw json with a text editor), we have also developed a webapp called the Orange Button Open API Editor for exactly that purpose.  You can find an instance of this editor that is free for you to use at the following URL: https://obeditor.sunspec.org
```

Comment:
```suggestion
You can visually explore and edit a copy of this json file (rather than editing the raw json with a text editor) using the Orange Button [Open API Editor](https://obeditor.sunspec.org).
```